stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -256,19 +256,30 @@ internal struct _ContiguousArrayBuffer<Element>: _ArrayBufferProtocol {
   @inlinable
   internal init(
     _uninitializedCount uninitializedCount: Int,
-    minimumCapacity: Int
+    minimumCapacity: Int,
+    growForAppend: Bool
```

Comment:
I think we'll need to add back the original two-param version of this initializer to preserve the ABI. (It can be declared `@usableFromInline internal` and obsoleted, though.)

Reply:
ah yeah, thank you